src/NzbDrone.Core/Download/CompletedDownloadService.cs

```diff
@@ -23,25 +24,37 @@ public class CompletedDownloadService : ICompletedDownloadService
         private readonly IEventAggregator _eventAggregator;
         private readonly IHistoryService _historyService;
         private readonly IDownloadedEpisodesImportService _downloadedEpisodesImportService;
+        private readonly IParsingService _parsingService;
+        private readonly Logger _logger;
 
         public CompletedDownloadService(IConfigService configService,
                                         IEventAggregator eventAggregator,
                                         IHistoryService historyService,
-                                        IDownloadedEpisodesImportService downloadedEpisodesImportService)
+                                        IDownloadedEpisodesImportService downloadedEpisodesImportService,
+                                        IParsingService parsingService,
+                                        Logger logger)
         {
             _configService = configService;
             _eventAggregator = eventAggregator;
             _historyService = historyService;
             _downloadedEpisodesImportService = downloadedEpisodesImportService;
+            _parsingService = parsingService;
+            _logger = logger;
         }
 
-        public void Process(TrackedDownload trackedDownload)
+        public void Process(TrackedDownload trackedDownload, bool ignoreWarnings = false)
         {
             if (trackedDownload.DownloadItem.Status != DownloadItemStatus.Completed)
             {
                 return;
             }
 
+            if (ignoreWarnings)
+            {
+                _logger.Info("Skipping ");
+                Import(trackedDownload);
```

Comment:
I think you should wrap all the check for warnings in this `if`, instead of short circuiting it,